api/src/main/java/org/openmrs/module/registrationcore/api/errorhandling/ErrorHandlingService.java

```diff
@@ -0,0 +1,6 @@
+package org.openmrs.module.registrationcore.api.errorhandling;
+
+public interface ErrorHandlingService {
```

Comment:
Would be nice to add some useful javadocs here

Reply:
added